app/views/hyku/api/v1/work/_work.json.jbuilder

```diff
@@ -1,104 +1,110 @@
 # frozen_string_literal: true
 # FIXME: many attributes here left nil so specs will pass
-json.uuid work.id
+json.cache! [@account.tenant, work.id, work.date_modified, work.member_of_collection_ids & collection_docs.pluck('id')] do
```

Comment:
Probably safest to switch `work.date_modified` to `work.solr_document['_version_']` like in the others above.